docs/application_profile.md

```diff
@@ -80,6 +80,7 @@ The DLME Prototype Application Profile a subset of the [Europeana Data Model](ht
 | agg_aggregated_cho | `edm:aggregatedCHO` | M           | 1...1       | reference   | generated by the application (implied by the existance of the solr doc) | Solr document represents aggregation; inferred 1:1 relationship with the ProvidedCHO instance. derive from data if possible |
 | agg_data_provider  | `edm:dataProvider`  | M           | 1...1       | literal     | The organization or entity that supplies data through a provider. | |
 | agg_data_provider_ar  | `edm:dataProvider`  | M           | 1...1       | literal     | The organization or entity that supplies data through a provider specified in the Arabic language. | |
+| agg_data_provider_collection  | `edm:dataProviderCollection`  | M           | 1...1       | literal     | The collection of which the resource is a part. | |
```

Comment:
I'm not seeing this property documented in the EDM (or dataProviderCountry, for that matter)? Do you have a reference for these handy?